indi-asi/CMakeLists.txt

```diff
@@ -5,25 +5,28 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
+find_package(ASI REQUIRED)
 find_package(CFITSIO REQUIRED)
 find_package(INDI REQUIRED)
 find_package(ZLIB REQUIRED)
 find_package(USB1 REQUIRED)
 find_package(Threads REQUIRED)
 
-# Using ASI Camera SDK Version 1.14.0715 updated on 2019-07-22
-# Using ASI EFW SDK Version 0.3.1205 updated on 2017-12-05
-# Using ASI ST4 SDK Version 1.0 updated on 2018-07-23
-# Using ASI EAF SDK Version 0.1.0524 updated on 2019-06-20
 set(ASI_VERSION_MAJOR 1)
-set(ASI_VERSION_MINOR 3)
+set(ASI_VERSION_MINOR 4)
+
+set(ASICAM /usr/lib/arm-linux-gnueabihf/libASICamera2.so)
+set(ASIEFW /usr/lib/arm-linux-gnueabihf/libEFWFilter.so)
+set(ASIST4 /usr/lib/arm-linux-gnueabihf/libUSB2ST4Conv.so)
+set(ASIEAF /usr/lib/arm-linux-gnueabihf/libEAFFocuser.so)
```

Comment:
Why are these hard-coded to arm?

Reply:
Good point. How should we pull the libs for various architectures? The present FindASI.cmake pull only lib for CCD. It should be using components to get all the libs. Can you handle it?

Reply:
Right components is the way to resolve this. I'll see if I have time today for this.

Reply:
I implemented it using much simpler approach than components. All libs are returned by findASI.cmake in ASI_LIBRARIES